launcher/main/main.go

```diff
@@ -41,12 +41,32 @@ func main() {
 		customConfigFile = os.Args[2]
 	}
 
-	cmd, err := launchlib.CompileCmdFromConfigFiles(staticConfigFile, customConfigFile)
+	// Read configuration
+	staticConfig, err := launchlib.GetStaticConfigFromFile(staticConfigFile)
+	if err != nil {
+		fmt.Println("Failed to read static config file", err)
+		panic(err)
+	}
+	customConfig, err := launchlib.GetCustomConfigFromFile(customConfigFile)
+	if err != nil {
+		fmt.Println("Failed to read custom config file", err)
+		panic(err)
+	}
+
+	// Create configured directories
+	err = launchlib.MkDirs(staticConfig.Dirs)
```

Comment:
Can combine call and check per comment further below, but purely a matter of preference.